org.spesmilo.Electrum.appdata.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+ <id>org.electrum.electrum</id>
+ <name>Electrum</name>
+ <developer_name>spesmilo</developer_name>
+ <summary>Lightweight Bitcoin client</summary>
+ <description>
+  <p>Main features are:</p>
+  <p>- Encrypted wallet: the file that contains bitcoin private keys is protected with a password, and never leaves the user's computer</p>
+  <p>- Deterministic key generation: If you lose your wallet file, you can recover it from its seed. You are protected from your own mistakes</p>
+  <p>- Instant on: by default the client does not download the blockchain, it requests that information from a server. No delays, always up-to-date</p>
+  <p>- Transactions are signed locally: Your private keys are not shared with the server. You do not have to trust the server with your money</p>
+  <p>- Cold storage: Keeping private keys offline is supported. Has a watch-only mode for online use</p>
+  <p>- Multisignature: Dividing the power to spend coins between multiple wallets is supported</p>
+  <p>- Hardware wallet integration: Many leading hardware wallets can interface with Electrum</p>
+  <p>- Redundancy: You are not tied to a particular server, and the server does not need to know you. One server going down doesn't cause user downtimes</p>
+  <p>- No single point of failure: The server code is open source, anyone can run a server. Private keys can be exported and imported into other wallets</p>
+  <p>- Firewall friendly: The client does not need to open a port, it simply polls the server for updates</p>
+  <p>- Free software: MIT License. Anyone can audit the code</p>
+  <p>- Written in Python. The code is short, and easy to review</p>
+  <p>- Add-ons: third-party plugins are supported</p>
+  <p>- Support for Bitcoin URIs, signed URIs and Bitcoin aliases.</p>
+ </description>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>MIT</project_license>
+ <url type="homepage">https://electrum.org/</url>
+ <screenshots>
+  <screenshot>https://en.bitcoin.it/w/images/en/f/fc/Capture-Electrum.png</screenshot>
+ </screenshots>
+ <categories>
```

Comment:
Don’t set categories in the AppData file.

https://github.com/flathub/flathub/wiki/AppData-Guidelines#icons-and-categories